core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.html

```diff
@@ -155,6 +155,7 @@
   }
 
   .conversation-skin-supplemental-card-container {
+    background-color: white;
```

Comment:
Please make sure to test other interactions that use the supplemental card container.

Reply:
Done, no conflicts.